README.md

```diff
@@ -1 +1,55 @@
-CI inter module management
+# ci
+
+The `ci` repository stores configuration files and libraries that are used in
+the Keep/tBTC Continous Integration process. It also contains the GitHub Actions
+workflow (`Main`) that acts as an entry point for inter-module builds.
+
+## Storing environment-specific variables
+
+Non-confidential, environment/network-specific variables used in some of the
+GitHub Actions workflows across `keep-network` repositories are stored in the
+`*.env` files under `config/env`. The `keep-network/load-env-variables` GH
+Action can be used to retrieve the data and load it to the workflow runner's
+context. Learn more from the Actions'
+[README](https://github.com/keep-network/load-env-variables/blob/main/README.md). 
+
+## Inter-module dependency management
+
+The `Main` workflow (accessible via `Actions` tab) acts as a single entry point
+for inter-module builds. Triggering this workflow via a workflow_dispatch event
+will start the chain of automatic executions of the workflows which
+build/publish Keep/tBTC modules, according to the order specified in the
+`config/config.json` file. Properly configured order allows the system to
+execute the downstream builds only after the upstream dependencies are built and
+published.
+
+To trigger the `Main` workflow, one needs to provide the following four inputs:
```

Comment:
Maybe it would be worth adding a screenshot showing where to find the button to dispatch a workflow run?